README.md

```diff
@@ -114,44 +114,46 @@ $ gem install fortnox-api
 # Usage
 
 ## Getting an AccessToken
-To make calls to the API server you need a `ClientSecret` and an `AccessToken`. When you sign up for an API-account with Fortnox you should get a client secret and an authorization code. To get the access token, that is reusable, you need to do a one time exchange with the API-server and exchange your authorization code for an access token. This can be done in several ways but we provide a method for it in the gem that you can use.
+To make calls to the API server you need a `ClientSecret` and an `AccessToken`. When you sign up for an API-account with Fortnox you should get a client secret and an authorization code. To get the access token, that is reusable, you need to do a one time exchange with the API-server and exchange your authorization code for an access token. For more information about how to get access tokens, see Fortnox developer documentation.
 
-> ​:warning: **Do not do this more than once!** If you try to do the auth code/access token exchange more than once, regardless of method, it will lock your API-account! So if you get the token using curl or whatever do not use this method as well. If your account is not working and you think it might be due to this you will have to contact Fortnox support and have them reset the authorization code for you. If you want to use several access tokens, you need to use a new authorization code for each one of them!
+## Configuration
+To configure the gem you can use the `configure` block. A `client_secret` and `access_tokens` are required configurations for the gem to work so at the very minimum you will need something like:
 
 ```ruby
-# Load the special class from the gem. You need to install the gem first ofc.
-require 'fortnox/api/access_token'
-
-Fortnox::API::AccessToken.new(
-  base_url: 'https://api.fortnox.se/3',
-  client_secret: 'P5K5wE3Kun', # Replace with your client secret
-  authorization_code: 'ea3862b1-189c-464b-8e25-1b9702365fa1', # Replace with your auth code
-)
+Fortnox::API.configure do |config|
+  config.client_secret = 'P5K5wE3Kun'
+  config.access_tokens = ['3f08d038-f380-4893-94a0a0-8f6e60e67a']
```

Comment:
Can we get this to be `config.access_token = '3f08d038-f380-4893-94a0a0-8f6e60e67a'` and just alias it over to `config.access_tokens` in the setter? Not sure how much dry config will let you do, but it would be a lot nicer.

Reply:
I'll have a look. I had that in mind too, but I wanted you to take a look at the code first :)